snap/snapcraft.yaml

```diff
@@ -0,0 +1,35 @@
+name: hwctl
+base: core22
+version: '0.1' # 
```

Comment:
Shouldn’t it match the project version?

Reply:
Best to drive all places where we refer to the version from git tags.